demo/src/test/java/com/example/test/core/datadriven/dataJson.java

```diff
@@ -0,0 +1,13 @@
+package com.example.test.core.datadriven;

```

Comment:
should change the dataJson name to other

Reply:
change dataJson name to dataJsonforStyleAndCategoryPage.java